apps/prairielearn/src/sprocs/authz_assessment.sql

```diff
@@ -7,6 +7,7 @@ CREATE FUNCTION
         OUT authorized boolean,      -- Is this assessment available for the given user?
         OUT exam_access_end timestamptz, -- If in exam mode, when will access end?
         OUT credit integer,          -- How much credit will they receive?
+        OUT end_date timestamptz,
```

Comment:
Can you explain why adding this new output parameter was needed?

Reply:
It's not required, It was done a while ago when I was still trying different possibilities of changing the assessment access rules. I have removed it now.